package.json

```diff
@@ -319,7 +319,7 @@
         "yargs": "^15.1.0"
     },
     "peerDependencies": {
-        "@box/cldr-data": "^34.2.0",
+        "@box/cldr-data": "^34.4.1",
```

Comment:
technically another breaking change, unless it'll work on the older version

Reply:
With the old version, it will return the right language names but they will come out unsorted. (Or sorted by their English name, more specifically.)

Reply:
@j3tan What do you think? Does that means I should amend a commit to put "breaking change" or not?